comit/src/asset/ethereum/erc20.rs

```diff
@@ -50,6 +50,12 @@ impl FromWei<U256> for Erc20Quantity {
     }
 }
 
+impl From<BigUint> for Erc20Quantity {
+    fn from(quantity: BigUint) -> Self {
+        Erc20Quantity(quantity)
+    }
+}
```

Comment:
:mega: This breaks encapsulation of the `Erc20Quantity` type because it allows construction of it without checking the invariant that quantity must not exceed `max_value`!